.devcontainer/devcontainer.json

```diff
@@ -17,7 +17,7 @@
             ]
         }
     },
-    "postCreateCommand": "python3 -m pip install -e  src",
+    "postCreateCommand": "python3 -m pip install -e  src && git pull",
```

Comment:
Let us do git pull first to make sure, we will pick up all the new dependencies
`git pull && python3 -m pip install -e  src`